src/components/file-preview.vue

```diff
@@ -123,21 +158,37 @@ export default {
         },
         initOptions: {
           background: false,
-          viewMode: 0,
+          viewMode: 1,
           autoCropArea: 1,
           zoomable: false
         }
       }
     };
   },
-  watch: {
-    "image.cropRatio"(newValue) {
-      this.setAspectRatio(newValue);
-    }
-  },
   computed: {
+    componentId() {
+      return "file-preview-" + ++id;
+    },
+    moreOptions() {
+      const moreOptions = Object.assign({}, this.options);
```

Comment:
I've had some troubles where Object.assign sometimes doesn't get rid of the reactivity of the data from Vuex. Relying on lodash's .clone is a safer bet

```suggestion
      const moreOptions = _.clone(this.options);
```